src/hooks/useOurMedia/useOurMedia.tsx

```diff
@@ -0,0 +1,28 @@
+import { useMediaQuery } from '@mui/material';
+
+import { theme } from './theme';
+
+type Breakpoints =
+    | 'phone'
+    | 'tablet'
+    | 'tabletLandscape'
+    | 'laptop'
+    | 'desktop'
+    | 'desktopWide';
+
+type MinOrMax = 'min' | 'max';
+
+export const useOurMedia = (
+    breakpoint: Breakpoints = 'phone',
+    minOrMax: MinOrMax = 'max'
+) => {
+    let mediaQuery: string;
```

Comment:
we should close up this logic in useMemo IMO, that hook potentially would be used across whole application, so on each prop change etc. it would be calculated once again, that's just unnecessary